service/backend/pchain/block.go

```diff
@@ -299,62 +285,13 @@ func (b *Backend) getChainIDs(ctx context.Context) (map[string]string, error) {
 	return b.chainIDs, nil
 }
 
-func (b *Backend) getBlockDetails(ctx context.Context, index int64, hash string) (*indexer.ParsedBlock, error) {
```

Comment:
`getBlockDetails` makes two network calls:

1. first retrieve full block from pchain api just to pick its height
2. then with the height retrieved block and timestamp from indexer api

Just refactored this function to make a single network call